src/test/java/com/ezylang/evalex/parser/TokenizerStringLiteralTest.java

```diff
@@ -96,4 +97,20 @@ void testErrorUnmatchedQuoteOffset() {
         .isInstanceOf(ParseException.class)
         .hasMessage("Closing quote not found");
   }
+
+  @Test
+  void testSingleQuoteAllowed() throws ParseException {
+    assertThatThrownBy(() -> new Tokenizer("'hello'", configuration).parse())
+        .isInstanceOf(ParseException.class);
```

Comment:
Avoid putting multiple tests into a single test method.
Extract this exception test to a separate exception test method, also check for the correct exception message.